es6-persistence/README.md

```diff
@@ -0,0 +1,77 @@
+# ES6 Persistence
+
+This module provides ES6 persistence when indexing workflows and tasks.
+
+## Build
+
+In order to use the ES6, you must change the following files from ES5 to ES6:
+
+https://github.com/Netflix/conductor/blob/master/server/build.gradle#L29
+
+https://github.com/Netflix/conductor/blob/master/settings.gradle#L3
```

Comment:
Line number could change when new content is added to these files. Please  provide direct instructions of what needs to be changed. For eg:
In file: Server > build.gradle
`include 'client','common','contribs','core', 'es5-persistence','jersey'` should be updated to
`include 'client','common','contribs','core', 'es6-persistence','jersey'`.

Reply:
Done.